bublik/interfaces/api_v2/config.py

```diff
@@ -18,13 +18,31 @@
 from bublik.data.models import Config, ConfigTypes, GlobalConfigNames
 from bublik.data.serializers import ConfigSerializer
 
+from django_filters import rest_framework as filters
```

Comment:
The order of the import blocks is incorrect (will be fixed as a result of applying the linter).